applications/photo-asset-manager/SPECIFICATION.md

```diff
@@ -1,45 +1,51 @@
 # PAM tech spec
 
-This is the tech spec for the Photo Asset Management cross-service example. This doc details the specifics of the deliverable app.
+These are the technical specifications for the Photo Asset Management cross-service example, which was designed by the maintainers of this repository to showcase AWS services and SDKs.
```

Comment:
Which services and SDKs?

Reply:
Many services, many SDK's, all of which will be presented in the README.md per Brian's request.